app/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,8 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="title_settings">Settings</string>
     <string name="title_preferences">Preferences</string>
     <string name="title_privacy">Privacy</string>
+    <string name="title_privacy_policy">Privacy Policy</string>
+    <string name="title_terms_of_service">Terms Of Service</string>
```

Comment:
It looks like the Privacy section was broken in spanish prior to your pr and we missed it. values-es needs the equivalent of the above three lines.

Reply:
Should I use google translator or Just add them with todo to translate later by a native speaker?

Reply:
Actually you can see them already translated in the values-es in the <!-- Actions --> section

Reply:
will update, mistakenly closed this PR  by push from wrong branch, will resolve this